.github/workflows/release-npm.yml

```diff
@@ -37,16 +37,14 @@ jobs:
 
     steps:
       - name: Checkout source
-        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        uses: actions/checkout@v4
```

Comment:
[Security scanning](https://github.com/TBD54566975/web5-js/security/code-scanning) checks for pinned dependencies using hash so pin to a v4 using the commit hash of the GitHub action.

> For GitHub workflows used in building and releasing your project, pin dependencies by hash.